examples/AzureSDKDemoSwift/Source/CustomTabBarController.swift

```diff
@@ -30,10 +30,6 @@ import UIKit
 class CustomTabBarController: UITabBarController {
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
-        do {
-            try StorageBlobClient.stopManaging()
-        } catch {
-            showAlert(error: error)
-        }
+        StorageBlobClient.stopManaging()
```

Comment:
I think there's one more I had found, in AppDelegate.swift there's a call to `try? StorageBlobClient.stopManaging()` that can have the try removed.